package.json

```diff
@@ -94,7 +94,21 @@
         "command": "fluttergpt.fixErrors.menu",
         "title": "Fix Errors",
         "when": "false"
+      },
+      {
```

Comment:
- Let's add the commands along with the other create commands above so they are properly grouped while reading.